webapp/components/survey/SurveyForm/nodeDefs/components/nodeDefTableCellHeader.js

```diff
@@ -10,21 +10,55 @@ import { useI18n } from '@webapp/store/system'
 import * as NodeDefUiProps from '../nodeDefUIProps'
 import NodeDefIconKey from './NodeDefIconKey'
 
+const LabelWithTooltip = (props) => {
```

Comment:
What do you think about extract this component?

Reply:
eheheh I didn't want to do it now but... it can be useful elsewhere... how can we call it?!

Reply:
I approved because yes maybe it is not the moment, the name is perfect in my opinion